src/components/tooltip/tooltip.js

```diff
@@ -1,12 +1,16 @@
 import Vue from '../../utils/vue'
 import ToolTip from '../../utils/tooltip.class'
 import warn from '../../utils/warn'
+import { getComponentConfig } from '../../utils/config'
 import normalizeSlotMixin from '../../mixins/normalize-slot'
 import toolpopMixin from '../../mixins/toolpop'
+import { HTMLElement } from '../../utils/safe-types'
```

Comment:
Same here, Could you move this up under `getComponentConfig`? we keep all the imports from the same directories together.